diff --git a/engine/rustdesk/flutter/lib/mobile/pages/remote_page.dart b/engine/rustdesk/flutter/lib/mobile/pages/remote_page.dart
index 565f6dc..f67c7fe 100644
--- a/engine/rustdesk/flutter/lib/mobile/pages/remote_page.dart
+++ b/engine/rustdesk/flutter/lib/mobile/pages/remote_page.dart
@@ -588,14 +588,14 @@ class _RemotePageState extends State<RemotePage> with WidgetsBindingObserver {
                 children: [
                   gFFI.ffiModel.pi.isSet.isTrue &&
                           gFFI.ffiModel.waitForFirstImage.isTrue
-                      ? emptyOverlay(MyTheme.canvasColor)
+                      ? emptyOverlay(kColorCanvas)
                       : () {
                           gFFI.ffiModel.tryShowAndroidActionsOverlay();
                           return Offstage();
                         }(),
                   _bottomWidget(),
                   gFFI.ffiModel.pi.isSet.isFalse
-                      ? emptyOverlay(MyTheme.canvasColor)
+                      ? emptyOverlay(kColorCanvas)
                       : Offstage(),
                 ],
               )),
@@ -619,7 +619,7 @@ class _RemotePageState extends State<RemotePage> with WidgetsBindingObserver {
                                 });
                               }
                               return Container(
-                                color: MyTheme.canvasColor,
+                                color: kColorCanvas,
                                 child: inputModel.isPhysicalMouse.value
                                     ? getBodyForMobile()
                                     : RawTouchGestureDetectorRegion(
@@ -770,7 +770,7 @@ class _RemotePageState extends State<RemotePage> with WidgetsBindingObserver {
   Widget getBodyForMobile() {
     final keyboardIsVisible = keyboardVisibilityController.isVisible;
     return Container(
-        color: MyTheme.canvasColor,
+        color: kColorCanvas,
         child: Stack(children: () {
           final paints = [
             ImagePaint(ffiModel: gFFI.ffiModel),
@@ -842,7 +842,7 @@ class _RemotePageState extends State<RemotePage> with WidgetsBindingObserver {
       }
     }
     return Container(
-        color: MyTheme.canvasColor, child: Stack(children: paints));
+        color: kColorCanvas, child: Stack(children: paints));
   }
 
   List<TTextMenu> _getMobileActionMenus() {
